buddyboss/buddyboss-platform
# Activity comments indentation for nested comments

### All Submissions:


Fixes #1627 

### How to test the changes in this Pull Request:

1. Make comment sub - comment
2. check sub comment spacing

### Proof Screenshots or Video


### Other information:
